packages/react/src/components/CalendarDayGrid/CalendarGridDayCell.tsx

```diff
@@ -222,10 +222,14 @@ export const CalendarGridDayCell: React.FunctionComponent<ICalendarGridDayCellPr
         !day.isInBounds && classNames.dayOutsideBounds,
         !day.isInMonth && classNames.dayOutsideNavigatedMonth,
       )}
-      ref={(element: HTMLTableCellElement) => {
-        customDayCellRef?.(element, day.originalDate, classNames);
-        day.setRef(element);
-      }}
+      ref={
+        isNavigatedDate
+          ? navigatedDayRef
+          : (element: HTMLTableCellElement) => {
+              customDayCellRef?.(element, day.originalDate, classNames);
+              day.setRef(element);
+            }
+      }
```

Comment:
`customDayCellRef` and the `day` ref both still need to be set even if the element is the currently navigated day. The way to do this would be to conditionally set `navigatedDayRef` within the current ref function:

```jsx
ref={(element: HTMLTableCellElement) => {
  customDayCellRef?.(element, day.originalDate, classNames);
  day.setRef(element);
  isNavigatedDate && navigatedDayRef.current = element;
}}
```

Then in line 18 of CalendarGridRow.tsx, change this:
`React.RefObject<HTMLButtonElement>`
to this:
` React.MutableRefObject<HTMLButtonElement>`